### PR TITLE
[WIP] Add GraphQL mutation to REST endpoints

### PR DESCRIPTION
**Pull request by mnlbox**
It's a PR for https://github.com/elitan/hasura-backend-plus/issues/15
Now it's just support `register` as mutation.
After some test we can add other endpoint as GraphQL mutation and use Hasura remote schema feature with it.

### CODE REVIEW

**Comment by mnlbox** [comment]
We must add `GRAPHQL_PORT` environment variable too.
Also Maybe it's better to add another environment variable as flag called `ENABLE_GRAPHQL` to able to enable or disable graphQL endpoint.
@elitan What is your idea?

File: src/graphql_wrapper/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+// replace with actual REST endpoint
+const restAPIEndpoint = 'http://localhost:3010';
+
+const resolvers = {
+    Query: {},
+    Mutation: {
+        register: async (_, { email, password } ) => {
+            const registerResponse = await postData(restAPIEndpoint + '/auth/register', { email, password } );
+            console.warn("++++++++" + registerResponse);
```
Comment (mnlbox) [inline src/graphql_wrapper/index.js]: @elitan Any idea why this message not printed?

Comment (elitan) [reply src/graphql_wrapper/index.js]: Can not see anything odd.

**Comment by elitan** [comment]
The main reason would be that you can not send files via the graphql protocol so the storage part of hasura-backend-plus need to be run as a express server (or similar).

**Pull request closed without merging**